client/lib/cart-values/cart-items.js

```diff
@@ -255,27 +254,23 @@ export function hasDomainCredit( cart ) {
  * @returns {Boolean} - Whether or not the cart contains a domain with that TLD
  */
 export function hasTld( cart, tld ) {
-	return some( getDomainRegistrations( cart ), function( cartItem ) {
-		return getDomainRegistrationTld( cartItem ) === '.' + tld;
+	return some( concat( getDomainRegistrations( cart ), getDomainTransfers( cart ) ), function(
+		cartItem
+	) {
```

Comment:
Stylistic nit -- the way the linter breaks this up slays me. Can we/should we do the concat into a temp var above the return?

Reply:
Eg: `const domainCartItems = concat( getDomainRegistrations( cart ), getDomainTransfers( cart ) );`